.changeset/early-crews-decide.md

```diff
@@ -0,0 +1,8 @@
+---
+"@justeattakeaway/pie-toggle-switch": minor
+"@justeattakeaway/pie-wrapper-react": minor
+"pie-storybook": minor
+"pie-monorepo": minor
+---
+
+[Added] - Created package skeleton for pie-toggle-switch component
```

Comment:
Didn't this happen as part of the previous PR (looks like a duplicate changeset?)

Reply:
Have deleted this now, not sure how this came in tbh.